docs/user-guides/configuration.md

```diff
@@ -145,31 +153,51 @@ The important parameters are the followings:
 - `agent.ngt.distance_type`
 - `agent.ngt.object_type`
 
-Users should configure these parameters first to fit to their use case.
+Users should configure these parameters first to fit their use case.
 
-For further details, please read [NGT wiki][yj-ngt-wiki].
+For further details, please read [the NGT wiki][yj-ngt-wiki].
 
-Agent-NGT has a feature to start indexing automatically.
+Vald Agent NGT has a feature to start indexing automatically.
 The behavior of this feature can be configured with these parameters:
 
 - `agent.ngt.auto_index_duration_limit`
 - `agent.ngt.auto_index_check_duration`
 - `agent.ngt.auto_index_length`
 
+<div class="notice">
+When Vald Agent NGT is indexing, all search requests to the target pod will be canceled.
+</div>
+
+<div class="warning">
+When deploying Vald Index Manager, the above parameters should be set much longer than the Vald Index Manager settings.
+(Please see Vald Index Manager section)<BR>
+This is because Vald Index Manager accurately grasps the index information of each Vald Agent NGT and controls the execution timing of indexing.<BR><BR>
```

Comment:
```suggestion
This is because the Vald Index Manager accurately grasps the index information of each Vald Agent NGT and controls the execution timing of indexing.<BR><BR>
```